app/domain/authentication/authn_jwt/orchestrate_jwt_authentication.rb

```diff
@@ -0,0 +1,46 @@
+require 'command_class'
+
+# This class is the starting point of the JWT authenticate requests, responsible to identify the vendor configuration and to run the JWT authenticator
+module Authentication
+  module AuthnJwt
+
+    OrchestrateAuthentication ||= CommandClass.new(
+      dependencies: {
+        validate_uri_based_parameters: Authentication::AuthnJwt::InputValidation::ValidateUriBasedParameters.new,
+        jwt_configuration_factory: Authentication::AuthnJwt::VendorConfigurations::ConfigurationFactory.new,
```

Comment:
this can be named `configuration_factory` to match the class name. it is obvious that it's part of `jwt`

Reply:
Done